ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2346,8 +2307,8 @@ <h3>Pay less and share the PLOG experience.</h3>
   <div class="row">
     <div class="span8 offset2">
       <p>One of the best parts of PLOG is sharing, even of the same room, because passion knows no schedules. Invite
-          your friends and your colleagues at PLOG and book a room for the whole of you, or visit the IRC channel and
-          find other people within the Plone community who are willing to pay less by sharing the same room
+          your friends and your colleagues to PLOG and book a room for the whole of you, or visit our forum and
```

Comment:
Good idea pointing no the forum (but maybe include the URL?). 

Change "for the whole of you" to "for all of you"?

Reply:
Sure I'll make it "visit our forum at community.plone.org, with a link.

I could change "for the whole of you"... I did change some of the Abstract-original Italianate English phrasings. But I find them a bit charming, and so Italian (and PLOG of course is very Italian) that I didn't want to native-English-speaker-ify all of them. Let me know if you feel strongly about this one.

Reply:
True, it is somewhat charming, but I wonder how many will understand that that is why you're leaving a distinctly non-native phrasing in there. Up to you!

Reply:
Added text and link to forum. And changed "for the whole of you", fair enough.